src/components/pages/Account/AccountHeader.tsx

```diff
@@ -0,0 +1,237 @@
+                <Avatar className={styles.image} />
+              )}
+              <div>
+                <h3 className={styles.name}>
+                  {name || accountTruncate(accountId)}
+                </h3>
+                {chainIds.map((value, index) => {
+                  const oceanConfig = getOceanConfig(value)
+                  if (!isShowMore && index > 0) return
+                  return (
+                    <ExplorerLink
+                      className={styles.truncate}
+                      networkId={value}
+                      path={`address/${accountId}`}
+                      key={value}
+                    >
+                      <code>
+                        {accountTruncate(accountId)} on{' '}
+                        {oceanConfig?.explorerUri}
+                      </code>
+                    </ExplorerLink>
+                  )
+                })}
+                {chainIds.length > 1 ? (
+                  <span className={styles.more} onClick={toogleShowMore}>
+                    {!isShowMore
+                      ? `Show ${chainIds.length - 1} more`
+                      : 'Show less'}
+                  </span>
+                ) : (
+                  ''
+                )}
+              </div>
+            </div>
+            <div className={styles.statisticsOverviewGrid}>
+              <Published className={styles.statisticsImages} />
+              <p className={styles.statisticsValues}>{numberOfAssets}</p>
+              <Sold className={styles.statisticsImages} />
+              <p className={styles.statisticsValues}>{sold}</p>
+              <Tvl className={styles.statisticsImages} />
+              <div className={styles.liquidity}>
+                <Conversion
+                  price={tvl?.price}
+                  className={styles.statisticsValues}
+                  hideApproximateSymbol
+                />
+                <Token symbol="OCEAN" balance={tvl?.oceanBalance} noIcon />
+              </div>
+              <p className={styles.statiscsLabel}>Published</p>
+              <p className={styles.statiscsLabel}>Sold</p>
+              <p className={styles.statiscsLabel}>TVL</p>
+            </div>
+          </div>
+          <div>
+            <p className={styles.descriptionLabel}>About</p>
+            <div>
+              <p id="description" className={styles.description}>
+                {description || 'No description found.'}
+              </p>
+              {isDescriptionTextClamped() ? (
+                <span className={styles.more} onClick={toogleShowMore}>
+                  <a
+                    href={`https://www.3box.io/${accountId}`}
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    Read more
+                  </a>
+                </span>
+              ) : (
+                ''
+              )}
+            </div>
+            <div className={styles.publisherLinks}>
```

Comment:
Displaying this section might be conditioned if there are any links to show. In my case there aren't any and the bullet item is empty: 

![Screenshot from 2021-08-23 14-30-41](https://user-images.githubusercontent.com/49017601/130441122-7e1124b4-7f06-4e3b-bdf1-8999ad70a0ad.png)